source/octf/proto/defs.proto

```diff
@@ -70,6 +70,23 @@ message LogTrace {
     string trace = 4;
 }
 
+message EventIoTimestamp {
+    uint64 sec = 1;
+    uint64 nsec = 2;
+};
+
+/**
+ * @brief Structure describing a given inode
+ *
+ * Combines inode file id with its creation date for more unambigious
+ * identification of specific file.
+ */
+message EventNodeId {
```

Comment:
Maybe FileId? And I'd move it to trace.proto

Reply:
Changed to FileNodeId - too many FIleIds floating around causing potential conflicts